### PR TITLE
fix ODF template

### PR DESCRIPTION
**Pull request by gayathrimenath-ibm**

### CODE REVIEW

File: config-templates/redhat/odf-remote/4.14/custom-parameters.json [hunk 1]
```diff
@@ -258,6 +258,6 @@
     "supported-values": ["true", "false"],
     "type": "boolean",
     "required": "true",
-    "mutable": "true"
+    "mutable": "false"
```
Comment (nkkashyap) [inline config-templates/redhat/odf-remote/4.14/custom-parameters.json:261]: I think this should be `mutable` 
it should be "mutable": "true"

**Review by nkkashyap — CHANGES_REQUESTED** [review]

File: config-templates/redhat/odf-remote/4.14/custom-parameters.json (hunk 1 above)
Comment (gayathrimenath-ibm) [reply config-templates/redhat/odf-remote/4.14/custom-parameters.json:261]: This parameter cannot be updated after installation

**Review by mssachan — APPROVED** [review]
LGTM

**Review by mssachan — APPROVED** [review]
lgtm

**Review by ashimagarg27 — APPROVED** [review]

**Review by nkkashyap — APPROVED** [review]
LGTM

**Pull request merged**